UPGRADE-2.0.md

```diff
@@ -39,3 +39,7 @@
     * The following configuration options were removed:
 
         * `sylius.mailer.templates`
+
+## Frontend
+
+* `use_webpack` option was removed from the `sylius_ui` configuration, and the Webpack has become the only module bundler provided by Sylius.
```

Comment:
It might be worth to explicitly mention removal of `use_webhook` twig global as anyone with extended templates might have it.